src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-9-16-0.mdx

```diff
@@ -0,0 +1,37 @@
+---
+subject: Ruby agent
+releaseDate: '2024-11-19'
+version: 9.16.0
+downloadLink: https://rubygems.org/downloads/newrelic_rpm-9.16.0.gem
+features: ["Instrumentation for aws-sdk-lambda", "Add new configuration options to attach custom tags (labels) to logs", "Update View Component instrumentation+"]
+bugs: ["Record explain plan traces on Rails 7.2+"]
+security: []
+---
+
+<Callout variant="important">
+  We recommend updating to the latest agent version as soon as it's available. If you can't upgrade to the latest version, update your agents to a version no more than 90 days old. Read more about [keeping agents up to date](/docs/new-relic-solutions/new-relic-one/install-configure/update-new-relic-agent/).
+
+  See the New Relic Ruby agent [EOL policy](https://docs.newrelic.com/docs/apm/agents/ruby-agent/getting-started/ruby-agent-eol-policy/) for information about agent releases and support dates.
+</Callout>
+
+## v9.16.0
+
+Version 9.16.0 introduces instrumentation for the aws-sdk-lambda gem, allows users to opt-in to adding labels to logs, updates View Component instrumentation, and fixes a bug with explain plans on Rails 7.2+.
+
+- **Feature: Instrumentation for aws-sdk-lambda**
+
+  If the aws-sdk-lambda gem is present and used to invoke remote AWS Lambda functions, timing and error details for the invocations will be reported to New Relic. [PR#2926](https://github.com/newrelic/newrelic-ruby-agent/pull/2926).
```

Comment:
When the aws-sdk-lambda gem is available and used to invoke remote AWS Lambda functions, the timing and error details of the invocations will be reported to New Relic. [PR#2926](https://github.com/newrelic/newrelic-ruby-agent/pull/2926).